src/chrome/content/rules/Github-Pages.xml

```diff
@@ -5,6 +5,28 @@
 
 	<target host="*.github.io" />
 
+
+	<!-- pages that require http -->
+	<exclusion pattern="^http://duckietv\.github\.io/DuckieTV/" />
+	<exclusion pattern="^http://ibotpeaches\.github\.io/Apktool/" />
+	<exclusion pattern="^http://schizoduckie\.github\.io/DuckieTorrent/" />
+	<exclusion pattern="^http://schizoduckie\.github\.io/PimpMyuTorrent/" />
+	<exclusion pattern="^http://leaflet\.github\.io/" />
+
+	<test url="http://duckietv.github.io/DuckieTV/" />
+	<test url="http://ibotpeaches.github.io/Apktool/" />
+	<test url="http://schizoduckie.github.io/DuckieTorrent/" />
+	<test url="http://schizoduckie.github.io/PimpMyuTorrent/" />
+	<test url="http://leaflet.github.io/Leaflet.markercluster/example/marker-clustering-realworld.388.html" />
```

Comment:
When greping though the repo I can still find some hardcoded `http:` links, but the initial issue seems to be fixed.